src/contribution.js

```diff
@@ -29,6 +29,12 @@ module.exports = (username, options) => {
     if (enableCors) url = `https://cors-anywhere.herokuapp.com/${url}`;
 
     https.get(url, response => {
+      if (response.statusCode !== 200) {
```

Comment:
This is breaking the tests which use CORS - I don't know why but it's returning a `400` when using the [cors-anywhere](https://cors-anywhere.herokuapp.com/) API. Could we make this check for `response.statusCode === 404` instead for now so that we can get this merged and I will investigate the other issue soon.

Also could we move this `if` statement into the start of the on response end callback, this will stop ESLint complaining, run `yarn lint` to check for linting errors.

Could you also please add some tests to test the promise rejection? If you could follow the existing format of 3 tests for each usage method that would be great!

If you need some help with writing tests then let me know, [here](https://jestjs.io/docs/en/api) are the docs to get you started!

Reply:
Sorry I forgot to test !
I will try to write the tests, thank's !

Reply:
Okay so I tried to write tests for promises and async/await (I don't really know how to catch with callback version ?)
And the tests with CORS enabled failed, I don't know why, if you have an idea ?
Sorry it's all new for me !
![preview.png](https://puu.sh/BPjdF/294955180e.png)